### DISCUSSION TITLE
Questions about pull request

### DISCUSSION THREAD

**Discussion by yuyenews** [TechEmpower/FrameworkBenchmarks#6366] [General]
Hello, I’m submitting a pull request for the first time and I don’t know much about the rules, so I would like to ask if there is any problem with this pull request. It has not been merged for 8 days. If there is any problem, you can give I write a comment, I will actively cooperate with the change.

https://github.com/TechEmpower/FrameworkBenchmarks/pull/6345

**Comment by codylerum** [comment]
I'm guessing it has to do with `PR: Please Update` label which is automatically added if a build fails, but is not removed once the build succeeds. #6368 has the same issue.
